### PR TITLE
OC-12443:pipe delimited was broken by other code change and refactor

### PR DESCRIPTION
**Pull request by Taoli2018**

### CODE REVIEW

File: src/main/java/core/org/akaza/openclinica/service/DataImportService.java [hunk 1]
```diff
@@ -333,8 +334,10 @@ public HashMap validateData(ODMContainer odmContainer, DataSource dataSource, Co
 
                         if (eventCRFStatus != null && eventCRFStatus.equals(DataEntryStage.INITIAL_DATA_ENTRY.getName())
                                 && eventCrfBean.getStatus().isAvailable()) {
+                        	eventCrfBean.setWorkflowStatus(EventCrfWorkflowStatusEnum.INITIAL_DATA_ENTRY);
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/service/DataImportService.java:337]: shouldn't this be handled by the markCRFStarted method?

File: src/main/java/core/org/akaza/openclinica/service/DataImportService.java [hunk 2]
```diff
@@ -333,8 +334,10 @@ public HashMap validateData(ODMContainer odmContainer, DataSource dataSource, Co
...
                             crfBusinessLogicHelper.markCRFStarted(eventCrfBean, userBean, true);
                         } else {
+                        	eventCrfBean.setWorkflowStatus(EventCrfWorkflowStatusEnum.COMPLETED);
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/service/DataImportService.java:340]: shouldn't this be handled by the markCRFComplete method?

Comment (Taoli2018) [reply src/main/java/core/org/akaza/openclinica/service/DataImportService.java]: done -- move to markCRFComplete method

File: src/main/java/core/org/akaza/openclinica/service/DataImportService.java (hunk 1 above)
Comment (Taoli2018) [reply src/main/java/core/org/akaza/openclinica/service/DataImportService.java]: done --  move to markCRFStarted method

**Comment by svadla-oc** [comment]
👍

**Pull request merged**